app/services/partner_profile_update_service.rb

```diff
@@ -0,0 +1,50 @@
+class PartnerProfileUpdateService
+  include ServiceObjectErrorsMixin
+  attr_reader :error
+  def initialize(old_partner, new_partner_params, new_profile_params)
+    @partner = old_partner
+    @old_profile = old_partner.profile
+    @profile = @partner.profile
+    @partner_params = new_partner_params
+    @old_served_areas = old_partner.profile.served_areas
```

Comment:
What are these `old_` variables used for?

Reply:
I believe there were there in memory of the ancestors of PartnerProfileUpdateService.  Tests work without them, so out they go in the next push.